packages/core/scss/typography/index.import.scss

```diff
@@ -0,0 +1,144 @@
+$kendo-font-size-md: $kendo-font-size !default;
+$kendo-font-size-lg: 1rem !default;
+$kendo-font-size-xl: 1.25rem !default;
+
+$kendo-line-height: k-math-div( 20, 14 ) !default;
+$kendo-line-height-xs: 1 !default;
+$kendo-line-height-sm: 1.25 !default;
+$kendo-line-height-md: $kendo-line-height !default;
+$kendo-line-height-lg: 1.5 !default;
+$kendo-line-height-em: calc( #{$kendo-line-height-md} * 1em ) !default;
+
+$kendo-font-weight: 400 !default;
+$kendo-font-weight-thin: 100 !default;
+$kendo-font-weight-extra-light: 200 !default;
+$kendo-font-weight-light: 300 !default;
+$kendo-font-weight-normal: $kendo-font-weight !default;
+$kendo-font-weight-medium: 500 !default;
+$kendo-font-weight-semibold: 600 !default;
+$kendo-font-weight-bold: 700 !default;
+
+$kendo-letter-spacing: null !default;
+$kendo-letter-spacing-tightest: -.15px !default;
+$kendo-letter-spacing-tighter: -.10px !default;
+$kendo-letter-spacing-tight: -.5px !default;
+$kendo-letter-spacing-normal: 0px !default;
+$kendo-letter-spacing-wide: .5px !default;
+$kendo-letter-spacing-wider: .10px !default;
+$kendo-letter-spacing-widest: .15px !default;
+
+$kendo-font-family-sans: Arial, Verdana, Tahoma, "Trebuchet MS", Helvetica, Impact, Gill Sans !default;
+
+$kendo-font-family-serif: "Times New Roman", Georgia, Garamond, Palatino, Baskerville !default;
+
+$kendo-font-family-sans-serif: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", "Noto Sans", "Liberation Sans", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !default;
+
+$kendo-font-family-monospace: SFMono-Regular, Menlo, Monaco, Consolas, "Roboto Mono", "Ubuntu Mono", "Lucida Console", "Courier New", monospace !default;
+
+$kendo-font-family: inherit !default;
+
+$_default-font-sizes: (
+    xs: $kendo-font-size-xs,
+    sm: $kendo-font-size-sm,
+    md: $kendo-font-size-md,
+    lg: $kendo-font-size-lg,
+    xl: $kendo-font-size-xl
+) !default;
+
+$_default-line-heights: (
+    xs: $kendo-line-height-xs,
+    sm: $kendo-line-height-sm,
+    md: $kendo-line-height-md,
+    lg: $kendo-line-height-lg,
+) !default;
+
+$_default-font-weights: (
+    light: $kendo-font-weight-light,
+    normal: $kendo-font-weight-normal,
+    medium: $kendo-font-weight-medium,
+    semibold: $kendo-font-weight-semibold,
+    bold: $kendo-font-weight-bold
+) !default;
+
+$_default-letter-spacings: (
+    tightest: $kendo-letter-spacing-tightest,
+    tighter: $kendo-letter-spacing-tighter,
+    tight: $kendo-letter-spacing-tight,
+    normal: $kendo-letter-spacing-normal,
+    wide: $kendo-letter-spacing-wide,
+    wider: $kendo-letter-spacing-wider,
+    widest: $kendo-letter-spacing-widest
+) !default;
+
+$_default-font-families: (
+    sans: $kendo-font-family-sans,
+    serif: $kendo-font-family-serif,
+    sans-serif: $kendo-font-family-sans-serif,
+    monospace: $kendo-font-family-monospace
+) !default;
+
+$kendo-font-sizes: $_default-font-sizes !default;
```

Comment:
A comment should be added so the Docs can pick it up:

```scss
/// The global default Font Sizes map.
/// @group typography-system
```

This should be done for all the maps and in each theme as well.